src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -100,6 +100,75 @@ public void start() {
       addBombardmentSources();
       m_needToAddBombardmentSources = false;
     }
+    
+    Territory lastAmphib = null, lastNonAmphib = null, t;
+    IBattle battle;
+    Iterator<Territory> battleTerritories = m_battleTracker.getPendingBattleSites(true).iterator(); // get bombing/air raids
+    if( battleTerritories.hasNext() ) {
+      t = battleTerritories.next();
+      if( battleTerritories.hasNext() ) return; // If more than one air/boming raid do nothing here
+      battle = m_battleTracker.getPendingBattle(t, true, null);
+      if( battle == null ) {
+        System.out.println("Air/Bombing Raid gone missing in BattleDelegate");
```

Comment:
No System.out.println's in production code please (should not really be in test code either).